sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -148,19 +148,36 @@ public AutoConfiguredOpenTelemetrySdkBuilder setResultAsGlobal(boolean setResult
     return this;
   }
 
+  /** Sets the {@link ClassLoader} to be used to load SPI implementations. */
+  public AutoConfiguredOpenTelemetrySdkBuilder setServiceClassLoader(
+      ClassLoader serviceClassLoader) {
+    requireNonNull(serviceClassLoader, "serviceClassLoader");
+    this.serviceClassLoader = serviceClassLoader;
+    return this;
+  }
+
   /**
    * Returns a new {@link AutoConfiguredOpenTelemetrySdk} holding components auto-configured using
    * the settings of this {@link AutoConfiguredOpenTelemetrySdkBuilder}.
    */
   @SuppressWarnings("deprecation") // Using classes which will be made package-private later.
   public AutoConfiguredOpenTelemetrySdk build() {
+    if (!customized) {
```

Comment:
this is just to make sure that if the builder is re-used, we don't SPI and customize more than once?

Reply:
Yeah